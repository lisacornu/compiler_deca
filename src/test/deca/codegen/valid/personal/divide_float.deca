// Description:
//    ecriture du résultat d'une opération
//
// Resultats:
//    2.0
//
// Historique:
//    cree le 07/01/2025

{
    float a = 4.0;
    float b = 2.0;
    println(a / b);
}
