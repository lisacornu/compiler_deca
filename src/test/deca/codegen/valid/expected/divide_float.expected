2.00000e+00
0.75000e+00
